pkg/cmd/cli/cmd/logs.go

```diff
@@ -189,5 +218,31 @@ func (o OpenShiftLogsOptions) RunLog() error {
 		// Use our own options object.
 		o.KubeLogOptions.Options = o.Options
 	}
-	return o.KubeLogOptions.RunLogs()
+	if !o.JenkinsLogs {
+		return o.KubeLogOptions.RunLogs()
+	} else {
+		bc, ok := o.KubeLogOptions.Object.(*buildapi.BuildConfig)
+		var urlString string
+		if !ok {
+			build, ok := o.KubeLogOptions.Object.(*buildapi.Build)
+			if ok {
+				urlString, _ = build.Annotations[buildapi.BuildJenkinsBlueOceanLogURLAnnotation]
+			}
+		} else {
+			buildName := buildutil.BuildNameForConfigVersion(bc.ObjectMeta.Name, int(bc.Status.LastVersion))
+			build, _ := o.Client.Builds(o.Namespace).Get(buildName, metav1.GetOptions{})
+			if build != nil {
+				urlString, _ = build.Annotations[buildapi.BuildJenkinsBlueOceanLogURLAnnotation]
+			}
+		}
+		if len(urlString) > 0 {
+			o.KubeLogOptions.Out.Write([]byte("Because this is a Jenkins Pipeline Strategy build, try accessing this URL:"))
```

Comment:
"Because this is a Pipeline Strategy build, the logs are accessed via the Jenkins server:"